test/Macros/macro_plugin_basic.swift

```diff
@@ -21,9 +20,9 @@ func test() {
 }
 
 //--- plugin
-#!/usr/bin/env python3
+#!PYTHON_EXEC_PATH
```

Comment:
Unfortunately I didn't think of this earlier, but using hashbang means that the mock tests will never work on Windows. The only way around that would be to create an actual executable though I think. At least this gets the test working for now, but would be nice if we could come up with something else here.